app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -19,29 +19,6 @@ def create
       render(json: education_benefits_claim)
     end
 
-    # TODO: This is hidden behind a flag in the routes, but it's duplicated here for some
```

Comment:
Sorry, a little confused by this comment. It's duplicated here, but where is the other place where this code lives? Is this comment solely for the `show` action? If so, why are we deleting that `daily_file` action as well? Does that live somewhere else too?